hachoir/wx/hex_view/hex_view.py

```diff
@@ -1,4 +1,5 @@
 import wx
+import darkdetect
```

Comment:
Would it be possible to catch ImportError and assume that the dark mode is disable if the module is missing?

Also, it would be nice to document the new dependency somewhere.

Currently, there is no doc for hachoir-wx. Maybe add a very short page in doc/?